pkg/monitoring/configuration/prometheus.go

```diff
@@ -0,0 +1,57 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2023 Red Hat, Inc.
+ *
+ */
+
+package configuration
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const emulationEnabledGauge = "emulationEnabledGauge"
+
+var (
+	Metrics = map[string]prometheus.Opts{
```

Comment:
Why is this public?

Reply:
leftover for what I thought was needed for the docs, changed